supabase/migrations/20260814230000_f010b_rls_lockdown.sql

```
@@ -0,0 +1,150 @@
+    to authenticated
+    using (
+        employee_id in (select e.id from public.employees e where e.profile_id = auth.uid())
+        or exists (select 1 from public.profiles p where p.id = auth.uid() and p.role = 'MANAGER')
+    );
+
+comment on policy "payroll_records_select_own_or_manager" on public.payroll_records is
+    'F-010-B: payroll record reads limited to the owning employee or managers.';
+
+-- 5. Payroll runs, benefits, departments, time_entries: any authenticated read.
+alter table public.payroll_runs enable row level security;
+drop policy if exists "Anyone can view payroll runs" on public.payroll_runs;
```

SQLFluff (4.2.2)

[error] Do not use special characters in identifiers.

(RF05)